packages/core/admin/package.json

```diff
@@ -52,7 +52,7 @@
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@pmmmwh/react-refresh-webpack-plugin": "0.5.7",
     "@strapi/babel-plugin-switch-ee-ce": "4.5.0",
-    "@strapi/design-system": "1.2.7",
+    "@strapi/design-system": "1.3.0-beta.2",
```

Comment:
I'm assuming if all these tests pass we can release the stable and use that instead?

Reply:
If the integration with beta works yes, we can release a stable version of the DS and just use it inside this branch and then release this branch